src/utils/defaultLanguageStrings.ts

```diff
@@ -329,6 +329,8 @@ export default {
   REVIEW_SUBMISSION_CONFIRM_OPTION: 'Submitting Review',
   REVIEW_SUBMISSION_FAIL: 'Not all sections have been reviewed',
   REVIEW_SUBMISSION_ERROR: 'Something wrong with server - review mutation failed',
+  REVIEW_SUBMISSION_INCOMPLETE_MESSAGE:
+    'You are about to submit your review. Some sections have not be completed by others, which means your decision is not changing the outcome - if aplicable. Are you sure you wish to proceed?',
```

Comment:
Oh, I only meant - In case approving our change the Outcome of the application. But if you think that's confusing I can rephrase. I'll try something better now.

Reply:
Done in [bf41f47](https://github.com/openmsupply/conforma-web-app/pull/1391/commits/bf41f4735257476f5047c04b984e9d3e2c7706b1)

Please have a look see if it's a bit better now

Reply:
Cool, maybe just a little more concise (and small grammar tweaks):
"You are about to submit your review, although your decision will not change the outcome yet as some sections are not yet completed by other reviewers. Are you sure you wish to proceed?"

Reply:
Done [0f11154](https://github.com/openmsupply/conforma-web-app/pull/1391/commits/0f1115484133adf7f3974c0730518dd66ddcd808)